suricata.yaml.in

```diff
@@ -1328,6 +1329,10 @@ decoder:
   # as it will sometimes detect non-teredo as teredo.
   teredo:
     enabled: true
+    # ports to look for Teredo. Max 4 ports. If no ports are given, or
+    # the value is set to 'any', Teredo detection runs on _all_ UDP packets.
+    ports: $TEREDO_PORTS # syntax: '3544, 1234'
```

Comment:
Or is the syntax "[3544, 1234]" like other lists that are specified as vars?

Reply:
You can use:
`"[1, 2, 3, 4]"`

`1, 2, 3, 4`

`"1,2,3,4"`

`"$TEREDO_PORTS,2,3,4"` (assuming that the var is defined)